components/script/dom/bindings/inheritance.rs

```diff
@@ -53,6 +53,7 @@ pub trait Castable: IDLInterface + DomObject + Sized {
     }
 }
 
+#[allow(missing_docs)]
```

Comment:
It would be nice to add documentation to this trait.

Reply:
Per https://github.com/servo/servo/pull/21105 I think this is only used for parent assertions by codegen/derive.